pkg/services/sqlstore/migrations/panel_mig.go

```diff
@@ -0,0 +1,35 @@
+package migrations
+
+import (
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func addPanelMigrations(mg *migrator.Migrator) {
+	mg.AddMigration("create panel table", migrator.NewAddTableMigration(panelv1()))
```

Comment:
is it maybe possible to create the table with the right columns and index directly in the `create table` statement instead of `alter`ing them or adding the index later?